### ISSUE TITLE
Adding accessibleName attribute for TimePicker component

### ISSUE THREAD

**Issue by dhruvojha** [SAP/ui5-webcomponents#6834]
### Is your feature request related to a problem?

The screen reading does not read labels for TimePicker components. This creates an accessiblity issue as screen reader are able to read other labels but not the TimePicker ones.

### Describe the solution you'd like

Need accessibleName attribute for TimePicker component to allow screen reader to read the label of the TimePicker, as it is able to do for other components.

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by petyabegovska** [comment]
The issue is duplicated to: https://github.com/SAP/ui5-webcomponents/issues/4686, so I close it.
